test/extern/fnPtrs/passBadFnsToCChapel-cPtrToChapel.good

```diff
@@ -0,0 +1,2 @@
+passBadFnsToCChapel.chpl:11: In function 'baz':
+passBadFnsToCChapel.chpl:12: error: Can't call a C function pointer within Chapel
```

Comment:
I don't personally think it's worth that.  I think if users want the feature, they'll ask for it and then we can think about whether or not we would be able to support it.  I don't think of the current test as "locking a behavior in" (in the sense that we couldn't/wouldn't change it), but rather making sure that our attempt to say something helpful in the absence of such a feature doesn't accidentally go away.  Put another way, the behavior of this test was much worse before I made some effort to generate this message and I don't want that effort to go away unobserved.  Nor do I want to preserve it by making it part of ".bad" output for a future test.